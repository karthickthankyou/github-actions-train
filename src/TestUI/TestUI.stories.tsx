import React from 'react'
import { Story, Meta } from '@storybook/react/types-6-0'
import TestUI, { ITestUIProps } from './TestUI'

export default {
  title: 'Pages/TestUI',
  component: TestUI,
} as Meta

const Template: Story<ITestUIProps> = (args) => <TestUI {...args} />

export const Primary = Template.bind({})
Primary.args = {}
