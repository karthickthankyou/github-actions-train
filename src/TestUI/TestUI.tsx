export interface ITestUIProps {}

const TestUI = ({}: ITestUIProps) => {
  return (
    <div>
      Hello, This is TestUI component. Updated with this text.
      <span>This is so cool!</span>
      <div
        style={{
          height: '200px',
          width: '200px',
          padding: '20px',
          background: '#ffff00',
        }}
      >
        Hello World
      </div>
    </div>
  )
}

export default TestUI
