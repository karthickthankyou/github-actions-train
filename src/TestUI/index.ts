import TestUI from './TestUI'
export default TestUI
