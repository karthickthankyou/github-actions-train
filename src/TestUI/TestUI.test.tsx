import React from 'react'
import { render } from '@testing-library/react'
import TestUI from './TestUI'

describe('TestUI Component', () => {
  test('it should match the snapshot', () => {
    const { asFragment } = render(<TestUI />)
    expect(asFragment()).toMatchSnapshot()
  })
})
