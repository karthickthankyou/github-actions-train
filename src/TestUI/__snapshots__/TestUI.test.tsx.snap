// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestUI Component it should match the snapshot 1`] = `
<DocumentFragment>
  <div>
    Hello, This is TestUI component. Updated with this text.
    <span>
      This is so cool!
    </span>
    <div
      style="height: 200px; width: 200px; padding: 20px; background: rgb(255, 255, 0);"
    >
      Hello World
    </div>
  </div>
</DocumentFragment>
`;
